03.macos/05.solving-problems/08.protect-mail-activity/docs.en.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Protect Mail Activity and AdGuard'
+taxonomy:
+    category:
+        - docs
+---
+
+## In a nutshell
+
+Apple's Mail app now uses a proxy to hide a user's IP Address when downloading images from emails.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/MAC/mac_protectMailActivity.png"> 
+
+
+However, it won't work if there's an active VPN connection. As it treats AdGuard as a VPN, it won't preload images automatically.
+
+Apple explains this issue [here](https://support.apple.com/en-us/HT212797).
+
+## In detail
```

Comment:
@nataMai what do you think about copy/pasting this "In detail" section to the iCloud Private Relay article as well? Sounds relevant.